TACC/Core-Portal
# Task/TV3-93: V2 historical jobs (Update JobSubmission model in v2 first)

## Overview
Create a new field in the `JobSubmission` model to retain job detail data for v2.
In the future (6 months), this model should be dropped.

## Related

* [TV3-93](https://jira.tacc.utexas.edu/browse/TV3-93)

## Changes
On job submission & resubmission it attaches a the job detail information to the `data` field.

## Testing

1. Ensure job submission & resubmission works properly.
2. Ensure that the jobs have the correct `data` field.

## Notes
